libs/usb-mocking/README.md

```diff
@@ -0,0 +1,38 @@
+# USB Drive Mocking
+
+The `bash` scripts in this library are useful for mocking USB drives in
+`vxsuite`. The mock consists of a simple loop device backed by an `.img` file.
+The contained filesystem persists and its contents can be separately inspected
```

Comment:
how do you separately inspect and manipulate the contents? Is is just in /media/vx/usb-drive ? If so can you add that info to the readme

Reply:
added